externs/shaka/player.js

```diff
@@ -2115,6 +2097,36 @@ shaka.extern.LiveSyncConfiguration;
 shaka.extern.StreamingConfiguration;
 
 
+/**
+ * @typedef {{
+ *   forceHTTP: boolean,
+ *   forceHTTPS: boolean,
+ *   minBytesForProgressEvents: number
+ * }}
+ *
+ * @description
+ * The Networking's configuration options.
+ *
+ * @property {boolean} forceHTTP
+ *   If true, if the protocol is HTTPs change it to HTTP.
+ *   If both forceHTTP and forceHTTPS are set, forceHTTPS wins.
+ *   <br>
+ *   Defaults to <code>false</code>.
+ * @property {boolean} forceHTTPS
+ *   If true, if the protocol is HTTP change it to HTTPs.
+ *   If both forceHTTP and forceHTTPS are set, forceHTTPS wins.
+ *   <br>
+ *   Defaults to <code>false</code>.
```

Comment:
Well it's not ideal configuration. As we have an opportunity now, maybe we should squeeze those 2 configs into one, like `overrideProtocol: enum(string)`?